operating/build-time-network-policy-tools.adoc

```diff
@@ -53,10 +111,61 @@ Applying network policies to the cluster as part of the workload deployment usin
 You can follow a GitOps approach by submitting the generated policies using pull requests, providing the team an opportunity to review the policies before deploying them as part of the pipeline.
 ====
 
+include::modules/roxctl-netpol-generate-command-options.adoc[leveloffset=+2]
+
 //Connectivity mapping
 include::modules/connectivity-mapping-using-the-roxctl-netpol-connectivity-map-command.adoc[leveloffset=+1]
 
+[id="retrieving-connectivity-mapping-information-from-a-kubernetes-manifest-directory_{context}"]
+=== Retrieving connectivity mapping information from a Kubernetes manifest directory
+
+.Procedure
+
+* Run the following command to retrieve the connectivity mapping information:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl netpol connectivity map _<folder_path>_ [flags] <1>
+----
++
+<1> Specify the path to the folder, which can include sub-folders that contain YAML resources and network policies for analysis, for example, link:https://github.com/np-guard/netpol-analyzer/tree/main/tests/netpol-analysis-example-minimal[`netpol-analysis-example-minimal/`]. The command scans the entire sub-folder tree. Optionally, you can also specify parameters to modify the behavior of the command.
++
+For more information about optional parameters, see xref:../operating/build-time-network-policy-tools.adoc#roxctl-netpol-connectivity-map-command-options_network-policy-tools[roxctl netpol connectivity map command options].
++
+.Example output
++
+====
+[%autowidth,options="header"]
+|===
+
+|src |dst |conn
+
+|0.0.0.0-255.255.255.255
+|default/frontend[Deployment]
+|TCP 8080
+
+|default/frontend[Deployment]
+|0.0.0.0-255.255.255.255
+|UDP 53
+
+|default/frontend[Deployment]
+|default/backend[Deployment]
+|TCP 9090
+
+|===
+====
++
+You can also view this output in link:https://github.com/np-guard/netpol-analyzer/blob/main/tests/netpol-analysis-example-minimal/connlist_output.md[md format].
```

Comment:
I think this sentence can be removed ("You can also view...") , since the example right above it is already in md format.

Another option is to add instead the links to the text and csv output of this example.